hoot-core/src/main/cpp/hoot/core/visitors/MultilineStringMergeRelationCollapser.cpp

```diff
@@ -132,123 +132,158 @@ void MultilineStringMergeRelationCollapser::visit(const ElementPtr& e)
 
   _numProcessed++;
 
-  if (!e)
+  if (!element || element->getElementType() != ElementType::Relation)
```

Comment:
Made bug fix here to handle the case where a relation has a type but no actual type tag.